salt/utils/network.py

```diff
@@ -46,18 +47,23 @@
     pass
 
 
-_INTERFACES = {}
+class Interfaces:
+    def __init__(self, interfaces=None):
+        if interfaces is None:
```

Comment:
```suggestion
class Interfaces:
    __slots__ = ("interfaces",)
    
    def __init__(self, interfaces=None):
        if interfaces is None:
```
This creates a thinner class without an internal dictionary, which it really doesn't need

Reply:
Fancy